packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -223,7 +223,9 @@ export interface TableOptions extends SchemaOptions {
    * This property can only be set if `encryption` is set to `TableEncryption.CUSTOMER_MANAGED`.
    *
    * @default - If `encryption` is set to `TableEncryption.CUSTOMER_MANAGED` and this
-   * property is undefined, a new KMS key will be created and associated with this table.
+   * property is undefined, a new encryption key will be created and associated with this table.
+   * If encryption and this property are both undefined, then the table is encrypted with 
```

Comment:
```suggestion
   * If `encryption` and this property are both undefined, then the table is encrypted with 
```

Surround the property name with backticks (`) so that it gets formatted in monospaced font, and users recognize it as another property in the API.